objcts.js

```diff
@@ -0,0 +1,9 @@
+for(i=1; i<=15;i++){
```

Comment:
As far as this is concerned you are using the variable `i` without declaring it. If this program is run in strict mode. It will throw an error. You declare variables in JavaScript by using either the `var` or `let` or `const` keywords.
For example, your for loop will look like
```js
for(var number = 1; number <= 15; number++)
```
> Also, avoid using single variable names, always use descriptive variable names. Forexample, see what I did for my for loop first line